cmake/configs/nuttx_px4fmu-v2_lpe.cmake

```diff
@@ -3,3 +3,8 @@ include(cmake/configs/nuttx_px4fmu-v2_default.cmake)
 list(APPEND config_module_list
 	modules/local_position_estimator
 	)
+
+list(REMOVE_ITEM config_module_list
```

Comment:
Yeah, we were running out of memory on 1MB boards. This was sometime back though, and I think the configs have had memory optimizations since. I'll try keeping INAV in.